sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -81,7 +85,11 @@
 
 class ClosableOutputStream(OutputStream):
   """A Outputstream for use with CoderImpls that has a close() method."""
-  def __init__(self, close_callback=None):
+  def __init__(
+      self,
+      close_callback=None  # type: Optional[Optional[Callable[[bytes], None]]]
```

Comment:
Why the double Optional (here and elsewhere below)?

Reply:
mistake.  fixed.